### PR TITLE
Add api docs and gallery on Windows build and build docs on Appveyor

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: doc/make.bat [hunk 1]
```diff
@@ -41,6 +44,9 @@ if "%1" == "clean" (
 )
 
 if "%1" == "html" (
+	mkdir source/api
+	%PYTHON% tools/build_modref_templates.py 
+	cd source && %PYTHON% random_gallery.py
```
Comment (JDWarner) [inline doc/make.bat]: In the Makefile we also build the API for the `latex` target. Might be more elegant to have a separate conditional for this.

**Comment by blink1073** [comment]
LGTM: https://ci.appveyor.com/project/blink1073/scikit-image-qbl76/build/1.0.329#L7219

**Comment by stefanv** [comment]
Those artifacts are really neat!  Perhaps they've been there for a while, but I've only noticed them now.  How do we get a 64-bit build up and running?

**Comment by blink1073** [comment]
I just added a 64bit 3.4 target.

**Comment by blink1073** [comment]
I went back to a single build because:  The Py34 64bit version had an old pip and would not allow me to upgrade it (it did not allow the `--trusted-host` option).  The Py27 64bit version was failing with:

```
from ._texture import (_glcm_loop,
ImportError: cannot import name _multiblock_lbp
```

**Comment by stefanv** [comment]
I'll keep those patches in there, and then we can try them again in a while.

**Pull request merged**